packages/contracts/src.ts/index.ts

```diff
@@ -79,6 +79,8 @@ export interface PopulatedTransaction {
 export type EventFilter = {
     address?: AccountLike;
     topics?: Array<string|Array<string>>;
+    fromTimestamp?: string;
+    toTimestamp?: string;
```

Comment:
ethers does not have `from` / `end` block in the `EventFilter` and polling by block numbers works with `from` and `to`. Lets see how it is implemented and keep it consistent